pxr/usd/usdSkel/utils.h

```diff
@@ -595,6 +596,18 @@ UsdSkelSkinNormalsLBS(const GfMatrix3f& geomBindTransform,
                      TfSpan<GfVec3f> normals,
                      bool inSerial=false);
 
+/// Skin normals with _faceVarying_ interpolation using linear blend skinning.
+USDSKEL_API
+bool
+UsdSkelSkinFaceVaryingNormalsLBS(const GfMatrix3d& geomBindTransform,
+                                 TfSpan<const GfMatrix3d> jointXforms,
+                                 TfSpan<const int> jointIndices,
+                                 TfSpan<const float> jointWeights,
+                                 int numInfluencesPerPoint,
+                                 TfSpan<const int> faceVertexIndices,
+                                 TfSpan<GfVec3f> normals,
+                                 bool inSerial=false);
+
```

Comment:
For completeness and consistency with the other UsdSkelSkin* utility methods here, would it be possible to also provide an overload that takes GfMatrix3f transforms?